rpcs3/Emu/System.cpp

```diff
@@ -1084,7 +1109,7 @@ void Emulator::Load(bool add_only, bool force_global_config)
 			}
 
 			// Store /dev_bdvd/ location
-			games[m_title_id] = bdvd_dir;
+			games[m_title_id] = bdvd_dir + m_game_dir;
```

Comment:
Wait a second, there is no need to change it.

Reply:
why not? you won't be able to differentiate between the games in the collection in the gamelist if you don't save the sfo dir instead. or am i missing something?

Reply:
Knowing bdvd_dir and title_id should be enough to find corresponding subdirectory

Reply:
yes. you might be right. I'll trying to use the title_id in GetSfoDirFromGamePath

Reply:
ugh, we don't know the title id at that point :rip:

Reply:
I don't see the problem with game list. It scans for PS3_DISC.SFB anyway.

Reply:
I updated it. as you can see the code is now much more complex.
Idk if that is worth it tbh